tests/baseline/cases/direct-cgw.yml

```diff
@@ -0,0 +1,23 @@
+# A pushsource library testcase.
+#
+# This file was generated from a template.
+# To regenerate, run test_baseline.py with PUSHSOURCE_UPDATE_BASELINE=1.
+
+# URL of Source to test.
+url: "CGW:my/great/file.yaml"
+
+# Push items generated from above.
```

Comment:
The test fails right now, which makes sense due to some mismatches in the data below.  For example why would the name be custom-filename, or the dest be repo1.

You should be able to reproduce this test failure locally.  Then you should do what the failure message suggests: set environment variable PUSHSOURCE_UPDATE_BASELINE=1 and run the test again.  It should then update this file with the output, i.e. the items you get if you request a source of "CGW:my/great/file.yaml".  If that output looks correct, add the update into your PR.